src/DurableTask.AzureStorage/Partitioning/TableLease.cs

```diff
@@ -0,0 +1,80 @@
+﻿//  ----------------------------------------------------------------------------------
+//  Copyright Microsoft Corporation
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  ----------------------------------------------------------------------------------
+
+#nullable enable
+namespace DurableTask.AzureStorage.Partitioning
+{
+    using System;
+    using Azure;
```

Comment:
nit: sort using statements
```suggestion
    using Azure;
    using System;
```

Reply:
I don't know but this is the result after I applied sort using.

Reply:
Oh whoops, I didn't realize that, I just saw the original "sort using statements" request had not been marked as resolved so I assumed work still needed to take place. Thanks for clarifying, @cgillum.

Reply:
Oh, my bad. I will resolve it.